Automattic/wp-calypso
# Stats: Fixing week view for Subscribers

Related to #75926

## Proposed Changes

* updating the hook that is fetching data and adjusting the date format for the `week` period

## Testing Instructions



* Open the live branch for this PR
* Navigate to the traffic stats page at `/stats/day/:siteSlug`
* Add the query string `?flags=stats/subscribers-section` to enable the subscriber counts page
* Navigate to the Subscribers page and verify that switching the chart's period to a `week` results in data being loaded (previously it resulted in a blank chart)

## Pre-merge Checklist
